content/news/2023-01-25-foodborne-training/index.md

```diff
@@ -1,19 +1,27 @@
 ---
-title: 'New GTN Material: "Pathogen detection from (direct Nanopore) sequencing data using Galaxy - Foodborne Edition"'
-tease: "Our new GTN material has been published last week along with multiple up coming workshops"
-authors: "Engy Nasr"
+title: 'New GTN Material and Workshops under the topic: "Pathogen detection from (direct Nanopore) sequencing data using Galaxy - Foodborne Edition"'
+tease: "Our new GTN material has been published last month along with multiple up coming workshops"
+authors: Engy Nasr, Bérénice Batut
 date: "2023-02-01"
 hide_tease: false
 tags: [training]
 subsites: [global]
 ---
 
-Our new GTN material has been published last week along with multiple up coming workshops: ["Pathogen detection from (direct Nanopore) sequencing data using Galaxy - Foodborne Edition"](https://training.galaxyproject.org/training-material/topics/metagenomics/tutorials/pathogen-detection-from-nanopore-foodborne-data/tutorial.html)
+Food contamination with pathogens are a major burden on our society. Globally, they affect an estimated 600 million people a year and impact socioeconomic development at different levels. These outbreaks are mainly due to Salmonella spp. followed by Campylobacter spp. and Noroviruses.
 
+The evolution of techniques in the last decades has made possible the development of methods to quickly identify responsible pathogen using their **DNA and without prior isolation**. Long-read sequencing tehcniques like ONT make these methods even **easier and more practical to identify strains quicker and with fewer reads**.
 
+In this tutorial, using direct Nanopore sequencing data, we show how to
 
-## Training Idea
-A workflow has been developed under the [__EOSC-Life industry call__](https://www.eosc-life.eu/industrycall/) project for creating an **Accessible** and **scalable** workflow for foodborne pathogens detection, identification and tracking for Nanopore sequenced data. Using Galaxy plaform we were able to create a [__FAIR__](https://www.go-fair.org/fair-principles/) workflow to agnostically detect pathogens anywhere not only in food. A training material in the Metagenomics category in the [__GTN__](https://training.galaxyproject.org/) has been created to guide learners and users to use and adapt the workflow. 
+1. **agnostically detect pathogens** (what exactly is this pathogen and what is the degree of its severity (its virulence factors) from data extracted directly (without prior cultivation) from a potentially contaminated sample (e.g. food like chicken, cow, etc.,) and sequenced using Nanopore
+2. **compare different samples to track** the possible source of contamination
+
+Our new GTN material has been published last month along with multiple up coming workshops: ["Pathogen detection from (direct Nanopore) sequencing data using Galaxy - Foodborne Edition"](https://training.galaxyproject.org/training-material/topics/metagenomics/tutorials/pathogen-detection-from-nanopore-foodborne-data/tutorial.html)
```

Comment:
```suggestion
## A tutorial for pathogen detection from (direct Nanopore) sequencing data using Galaxy

Our tutorial material is available since last month on the Galaxy Training Network: ["Pathogen detection from (direct Nanopore) sequencing data using Galaxy - Foodborne Edition"](https://training.galaxyproject.org/training-material/topics/metagenomics/tutorials/pathogen-detection-from-nanopore-foodborne-data/tutorial.html)
```